sky/utils/controller_utils.py

```diff
@@ -45,7 +45,8 @@
     'Details:\n  {err}')
 
 # The placeholder for the local skypilot config path in file mounts.
-LOCAL_SKYPILOT_CONFIG_PATH_PLACEHOLDER = 'skypilot:local_skypilot_config_path'
+_LOCAL_SKYPILOT_CONFIG_PATH_SUFFIX_IDENTIFIER = (
```

Comment:
- update docstr
- is it just a suffix? remove "_id" part?

Reply:
Done.